src/cmdlib.sh

```diff
@@ -426,6 +426,9 @@ EOF
         # available in content_sets.yaml. The mapped repos are then available in content_manifest.json
         # Feature: https://issues.redhat.com/browse/GRPA-3731
         create_content_manifest "$configdir"/content_sets.yaml "${tmp_overridesdir}/contentsetrootfs/usr/share/buildinfo/content_manifest.json"
+        # adjust permissions to appease the ext.config.shared.files.file-directory-permissions test
+        chmod 0644 "${tmp_overridesdir}/contentsetrootfs/usr/share/buildinfo/content_manifest.json"
+        chmod 0755 "${tmp_overridesdir}/contentsetrootfs/usr/share/buildinfo/"
```

Comment:
Actually, this *should* be the default, so you might not even need this one. Obviously doesn't hurt.

Reply:
I'll double check

Reply:
![you are right](https://media.giphy.com/media/vDWRVg8ZsjNWB583HV/giphy.gif)